crates/ra_project_model/src/lib.rs

```diff
@@ -240,6 +241,11 @@ impl ProjectWorkspace {
                                 }
                             }
                         }
+                        if let Some(core) = libcore {
```

Comment:
Playing around a bit with this, just changing the dependency order here also seems to fix the prelude issue. `std` is picked up before `core`. However I think that's a less stable solution, since I'm not sure if the dependencies are really an ordered list which is guaranteed to be stable. It normally shouldn't be necessary, so not betting on it is likely better.

Reply:
I agree with Florian that we should just rely on dependency order here, rather than special case the core name